kim_convergence/scale.py

```
@@ -553,30 +550,34 @@ class RobustScale():
+        quantile_range: tuple[float, float] = (25.0, 75.0),
+    ):
         self.with_centering_ = with_centering
         self.with_scaling_ = with_scaling
 
-        if not isinstance(quantile_range, tuple) or \
-                not isinstance(quantile_range, list):
-            raise CRError(f'invalid quantile range: {str(quantile_range)}.')
+        if not isinstance(quantile_range, tuple) or not isinstance(
+            quantile_range, list
+        ):
+            raise CRError(f"invalid quantile range: {str(quantile_range)}.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)